docker-compose.yml

```diff
@@ -18,6 +18,20 @@ services:
       volumes:
         - redis-data:/data
 
+    extractor:
+      image: thedeep/deep-extractor:latest
+      build:
+        context: ./deepl-service/deepl-deep-integration/
+      command: bash -c '/code/scripts/run_develop.sh'
+      volumes:
+        - ./deepl-service/deepl-deep-integration/:/code
+      tty: true
+      env_file:
+        - .env
+      ports:
+        - '8001:8001'
+      hostname: extractor
```

Comment:
Defining hostname is redundant here as it will be the service name which is what the value is.